modules/end-user-guide/partials/proc_creating-workspaces.adoc

```diff
@@ -2,4 +2,215 @@
+            exposure: public
+            name: code-redirect-3
+            protocol: http
+            targetPort: 13133
+      name: vscode-runtime-description
+    - name: checode
+      volume:
+        ephemeral: true
+  events:
+    preStart:
+      - init-container-command
+----
+<1> Name of the `DevWorkspaceTemplate` custom resource. This name will be used to reference the `DevWorkspaceTemplate` custom resource within the `DevWorkspace` custom resource.
+<2> User namespace, which is the target {orch-namespace} for the new workspace.
+====
+
+. To prepare the `DevWorkspace` custom resource, copy the contents of the target Git repository's devfile.
++
+.Copied devfile contents with `schemaVersion: 2.2.0`
+====
+[source,yaml,subs="+quotes,+attributes"]
+----
+components:
+  - name: tooling-container
+    container:
+      image: quay.io/devfile/universal-developer-image:ubi8-latest
+----
+====
++
+TIP: For more details, see the link:https://devfile.io/docs/2.1.0/what-is-a-devfile[devfile v2 documentation].
+
+. Create a `DevWorkspace` custom resource, pasting the devfile contents from the previous step under the `spec.template` field.
++
+.A `DevWorkspace` custom resource
+====
+[source,yaml,subs="+quotes,+attributes"]
+----
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: my-devworkspace#<1>
+  namespace: user1-dev#<2>
+spec:
+  routingClass: che
+  started: true#<3>
+  contributions:#<4>
+    - name: ide
+      kubernetes:
+        name: che-code#<5>
+  template:
+    projects:#<6>
+      - name: my-project-name
+        git:
+          remotes:
+            origin: https://github.com/eclipse-che/che-docs
+    components:#<7>
+      - name: tooling-container
+        container:
+          image: quay.io/devfile/universal-developer-image:ubi8-latest
+----
+<1> Name of the `DevWorkspace` custom resource. This is will be the name of the new workspace.
+<2> User namespace, which is the target {orch-namespace} for the new workspace.
+<3> Determines whether the workspace must be started when the `DevWorkspace` custom resource is created.
+<4> Reference to the `DevWorkspaceTemplate` custom resource of the selected IDE.
+<5> Name of the `DevWorkspaceTemplate` custom resource from the previous step.
+<6> Details about the Git repository to clone into the workspace when it starts.
+<7> List of components such as workspace containers and volume components.
+====
+
+. Apply the `DevWorkspace` custom resource to the cluster.
+
+. Verify that the workspace is starting by checking the *PHASE* status of the `DevWorkspace`.
++
+[subs="+quotes,attributes"]
+----
+$ {orch-cli} get devworkspaces -n __<user_{orch-namespace}>__  --watch
+----
++
+.Output
+====
+[subs="+quotes,attributes"]
+----
+NAMESPACE            NAME                  DEVWORKSPACE ID             PHASE      INFO
+user1-dev	     my-devworkspace       workspacedf64e4a492cd4701   Starting   Waiting for workspace deployment
```

Comment:
It looks like you changed the namespace from something longer, breaking alignment on the table:

```suggestion
NAMESPACE        NAME                  DEVWORKSPACE ID             PHASE      INFO
user1-dev	     my-devworkspace       workspacedf64e4a492cd4701   Starting   Waiting for workspace deployment
```

Reply:
Ah actually it looks like it's a mix of tabs and spaces here (the original code looks different in the view vs in the PR comment :upside_down_face:)
```suggestion
NAMESPACE        NAME                  DEVWORKSPACE ID             PHASE      INFO
user1-dev        my-devworkspace       workspacedf64e4a492cd4701   Starting   Waiting for workspace deployment
```

Reply:
@amisevsk, :smile: that was me, sorry.